### ISSUE TITLE
[BUG]  issue with loading FW(FW timeout) from branch: Main + latest zephyr on MTL RVP  at all test

### ISSUE THREAD

**Issue by wszypelt** [thesofproject/sof#7433]
**Describe the bug**

issue with loading FW(FW timeout) from branch: Main + newest Zephyr on MTL RVP at all tests

![image](https://user-images.githubusercontent.com/82375849/231383615-2961b941-53f4-4782-89be-f7ad0c5a77c1.png)

TEST DESCRIPTION:
```
        TEST DESCRIPTION:
        Firmware loading test
        _____________________
        PARAMETERS:
        --fw or --build_path - path to firmware binary file. parameters passed in command line or pytest.ini file
        _____________________
        TEST STEPS:
        1. Load FW.
        2. Check if FW was loaded (if not return ROM status)
        3. If interactive is True, test will run interactive python shell
        _____________________
        ACCEPTANCE CRITERIA:
        ·    No errors during test.
        ·    FwReadyNotification received.
```
 
**To Reproduce**
pytest -s "tests/avs/fw_00_basic/test_00_load_fw_basic.py::TestLoadFwBasic::test_00_00_load_fw_default"
Reproduction Rate
100%

**Impact**
After Bisect:
Last passed: SOF_FW_MAIN_ZEPHYR_01.01.00.4365 28.03.2023 20:11
First failed: SOF_FW_MAIN_ZEPHYR_01.01.00.4369 29.03.2023 20:11

**Environment**

SOF: SOF_FW_MAIN_ZEPHYR_01.01.00.4369
Platform: MTL RVP

**Logs**
[fw_load.zip](https://github.com/thesofproject/sof/files/11208916/fw_load.zip)

**Comment by kv2019i** [comment]
I could reproduce this locally as well. FYI @nashif as well, binary built with Zephyr main no longer boots:

```
[1576745.567310] snd_sof_intel_hda_common:hda_dsp_cl_boot_firmware: sof-audio-pci-intel-mtl 0000:00:1f.3: Firmware download successful, booting...
[1576747.618410] sof-audio-pci-intel-mtl 0000:00:1f.3: ------------[ DSP dump start ]------------
[1576747.618452] sof-audio-pci-intel-mtl 0000:00:1f.3: Firmware boot failure due to timeout
[1576747.618469] sof-audio-pci-intel-mtl 0000:00:1f.3: fw_state: SOF_FW_BOOT_IN_PROGRESS (2)
[1576747.618515] sof-audio-pci-intel-mtl 0000:00:1f.3: ROM status: 0x5, ROM error: 0x0
[1576747.618532] sof-audio-pci-intel-mtl 0000:00:1f.3: ROM debug status: 0x50000005, ROM debug error: 0x0
[1576747.618554] sof-audio-pci-intel-mtl 0000:00:1f.3: ROM feature bit enabled
[1576747.618567] sof-audio-pci-intel-mtl 0000:00:1f.3: ------------[ DSP dump end ]------------
[1576747.618580] sof-audio-pci-intel-mtl 0000:00:1f.3: error: failed to boot DSP firmware -5
[1576747.618687] snd_sof:sof_set_fw_state: sof-audio-pci-intel-mtl 0000:00:1f.3: fw_state change: 2 -> 3
```

**Comment by kv2019i** [comment]
Fix submitter to Zephyr.

**Comment by wszypelt** [comment]
Fix checked on 8a1ba8e47afe00d03d6b04c872220069874632fe
no issue, i close the issue
